rpm/fedora-34/Dockerfile

```diff
@@ -0,0 +1,25 @@
+ARG GO_IMAGE
+ARG DISTRO=fedora
+ARG SUITE=33
```

Comment:
Shouldn't we bump this `SUITE` to `34`? 🤔

Reply:
☺️ 🤦 it helps to actually `git add` after updating 😂 

```bash
git status
On branch add_new_ubuntu_fedora
Your branch is up to date with 'origin/add_new_ubuntu_fedora'.

Changes not staged for commit:
  (use "git add <file>..." to update what will be committed)
  (use "git restore <file>..." to discard changes in working directory)
	modified:   rpm/fedora-34/Dockerfile
```

Reply:
Pushed 👍

Reply:
Thanks 👍